### PR TITLE
Modified pod-utilities Dockerfile and version in Makefile

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
Changing the Dockerfile for pod-utilities to use `go build` instead of the existing `bazel build`.
Bazel has not been offering stable output paths. Hence switching to go build.

Please view the below discussion to understand the issue faced while building images for pod-utilites using bazel:
https://ibm-systems.slack.com/archives/G01CH7AC8H1/p1611726443000700

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/121#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: images/pod-utilities/Dockerfile
Comment (mkumatag) [inline images/pod-utilities/Dockerfile:32]:
```suggestion
    && go build -o /usr/local/bin/${POD_UTILITY} ./prow/cmd/${POD_UTILITY}
```

you can directly use the -o option with `go build` command instead of an additional cp command

File: images/pod-utilities/Dockerfile
Comment (mkumatag) [inline images/pod-utilities/Dockerfile:13]: I'm wondering instead of using ubuntu whether we can use the `golang:1.15` image instead of downloading the golang tar?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: images/pod-utilities/Dockerfile
Comment (Rajalakshmi-Girish) [reply images/pod-utilities/Dockerfile:13]: yes, even I thought so. Shall verify and resubmit.

File: images/pod-utilities/Dockerfile
Comment (Rajalakshmi-Girish) [reply images/pod-utilities/Dockerfile]: Done.

File: images/pod-utilities/Dockerfile
Comment (Rajalakshmi-Girish) [reply images/pod-utilities/Dockerfile]: Verified and submitted.

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/121#issuecomment-769680584" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/121#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
